src/private/guitypes.m

```diff
@@ -0,0 +1,13 @@
+function types = guitypes()
+% GUITYPES returns a cell array of MATLAB/Octave GUI object types
+%
+% Syntax
+%   types = guitypes()
+%
+% These types are not ignored by matlab2tikz and figure2dot.
```

Comment:
What about this comment? The way I read the code, those types *are* ignored by m2t and f2d.